### DISCUSSION TITLE
Adding primary buttons

### DISCUSSION THREAD

**Discussion by everndah** [squidfunk/mkdocs-material#5649] [Q&A]
I want to add a button to a link in my landing page, but when trying to display a button using what has been provided in the documentation it does not turn into a button. 

This is the line that I have added to my .md file

`[Subscribe to our newsletter](#){ .md-button .md-button--primary }`

The display on my landing page looks as shown in the image below. 

![image](https://github.com/squidfunk/mkdocs-material/assets/66929420/c1ba4bc3-f998-4725-a197-109a85f0d255)

What have I not done?

**Comment by kamilkrzyskow** [comment]
This looks like a lack of the markdown extension in the MkDocs.yml

```yml
markdown_extensions:
  - attr_list
```

https://squidfunk.github.io/mkdocs-material/reference/buttons/#configuration

**Reply by everndah** [reply]
there is a markdown extension in the mkdocs.yml.

**Reply by kamilkrzyskow** [reply]
Then something else is causing this issue, either bad formatting or some sort of other markdown extension is interfering with parsing of the `attr_list`. Here is a minimal example which is properly parsed on my end using the latest `mkdocs-material` version.
[test-buttons.zip](https://github.com/squidfunk/mkdocs-material/files/11868814/test-buttons.zip)
